index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="flex-container header">
+        <div class="flex-item">
+            <a href="#">Portfolio</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">About</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">Contact</a>
+        </div>
+    </div>
+    <div class="flex-container main">
+        <div class="flex-box">
+            <h1 class="intro">Hey There. I'm Ahmad.</h1>
+            <h1 class="profession">I'm a Full-stack Web Developer.</h1>
+            <p>I can help you build interactive and responsive websites. To be sure about the quality of my work, you may view my portfolio.</p>
+        </div>
+        <div>
```

Comment:
Kindly implement the social media icons and links to your project using a `ul` tag with an `a` tag embedded in each `li` tag of the `ul` tag, so as to have a pixel-perfect design, and complete the requirements for this task. 😃 

- Your design
<img width="167" alt="wrongmain" src="https://user-images.githubusercontent.com/41163571/219193170-e2c5cc4f-5644-40d7-87b1-5ea39f44ac8f.png">

- Figma design
<img width="199" alt="righticons" src="https://user-images.githubusercontent.com/41163571/219193455-c3c1ef72-1b23-4cbf-8120-3aef6dd509c7.png">